File: Data Structures For Interview/Java/Matrix Programs/SubtractMatrix.java
class SubtractMatrix {

    static void sub(int m1[][],int m2[][]){
        int m3[][]=new int[3][3];

        for(int i=0;i<3;i++){
            for(int j=0;j<3;j++){
                m3[i][j]=m1[i][j]-m2[i][j];
            }
        }

        for(int i=0;i<3;i++){
            for(int j=0;j<3;j++){
                System.out.print(m3[i][j]+" ");
            }
            System.out.println();
        }


    }

    public static void main(String s[]){

        int m1[][]={
            {10,10,10},
            {10,10,10},
            {10,10,10}
        };

        int m2[][]={
            {1,2,3},
            {4,5,6},
            {7,8,9}
        };

        sub(m1,m2);

    }
}

File: Data Structures For Interview/JavaScript/Matrix Programs/SubtractMatrix.js
function sub(m1,m2){

    let m3=[
        [0,0,0],
        [0,0,0],
        [0,0,0]
    ];

    for(let i=0;i<m1.length;i++){
        for(let j=0;j<m1[0].length;j++){
            m3[i][j]=m1[i][j]-m2[i][j];
        }
    }
    
    console.log(m3);

}

let m1=[
    [10,10,10],
    [10,10,10],
    [10,10,10]
];

let m2=[
    [1,2,3],
    [4,5,6],
    [7,8,9]
];

sub(m1,m2)